docs/pages/access-controls/guides/locking.mdx

```diff
@@ -6,12 +6,12 @@ description: How to lock compromised users or nodes
 <Details
   title="Version warning"
   opened={true}
-  scope={["oss", "enterprise"]}
+  scope={['oss', 'enterprise']}
```

Comment:
Same as the other file, there are a bunch of needless formatting changes here. This makes it difficult spot genuine changes from formatting ones and shifts line history for no reason,

Please revert and update your plugins accordingly. Same for other files added to the PR.